<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(196, 181, 183);
background-color: rgb(208, 218, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 85, 127);
font: 700 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUpdate">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);


font: 700 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Father's Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Student Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 20pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; text-decoration: underline;&quot;&gt;Student Record Management System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(213, 186, 224);
border-top-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFather">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(213, 186, 224);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>District :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSearch">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);

font: 700 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDistrict">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(213, 186, 224);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelete">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 85, 127);
font: 700 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Caste :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Roll No :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCaste">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(213, 186, 224);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRoll">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(213, 186, 224);
border-top-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrevious">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 85, 127);
font: 700 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 85, 127);
font: 700 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGPA">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(213, 186, 224);
border-top-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);
font: 700 9pt &quot;Segoe UI&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>GPA :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
